modules/distr-tracing-rn-new-features.adoc

```diff
@@ -13,17 +13,48 @@ Result – If changed, describe the current user experience.
 
 This release adds improvements related to the following components and concepts.
 
+== New features and enhancements {DTProductName} 2.4
+
+This release of {DTProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
+
+This release also adds support for auto-provisioning certificates using the Red Hat Elasticsearch Operator.
+
+* Self-provisioning, which means using the {JaegerName} Operator to call the Red Hat Elasticsearch Operator during installation. Self provisioning is fully supported with this release.
+* Creating the Elasticsearch instance and certificates first and then configuring the {JaegerShortName} to use the certificate is a Technology Preview for this release.
+
+[NOTE]
+====
+When upgrading to Elasticsearch 2.4, the Operator recreates the Elasticsearch instance, which might take five to ten minutes.
```

Comment:
@JStickler 
`When upgrading to Elasticsearch 2.4` should be as `When upgrading to distributed tracing 2.4`.
Also needs to specify that the recreation period is a complete down time.